BEACON-V2-Model/biosamples/endpoints.json

```diff
@@ -305,6 +307,26 @@
           "$ref": "https://raw.githubusercontent.com/ga4gh-beacon/beacon-framework-v2/main/common/beaconCommonComponents.json#/definitions/Limit"
         }
       },
+      "ids": {
+        "name": "id",
+        "in": "query",
+        "schema": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+        },
+      },
+      "individualIds": {
```

Comment:
Whilst this is correct as a potential GET parameter we should probably move this out to be defined within the individuals model, this GET parameter will only be available if individuals is supported by the beacon. Each entryType will come with a list of possible GET parameters that can be used with all query endpoints so packaging the GET parameters with the entry type seems cleaner and allows much easier creation of custom models + GET parameters